monkey/monkey_island/cc/ui/src/components/utils/ServerUtils.tsx

```diff
@@ -61,6 +61,16 @@ export function getMachineHostname(machine): string {
   return hostname;
 }
 
+export function getMachineFromID(id, machines) {
+  for (let machine of machines) {
+    if (machine.id === id) {
+      return machine;
+    }
+  }
```

Comment:
Doing 
```
return machines.filter((machine) => machine.id === id)[0] || null
```
Would be more concise

Reply:
I'd prefer to leave it as is for readibility.